src/index.html

```diff
@@ -7,7 +7,7 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <div class="calendar calendar--start-day--tue calendar--month-length-31">
+    <div class="calendar calendar--start-day--tue calendar--month-length-30">
```

Comment:
read the [task](https://github.com/mate-academy/layout_calendar#:~:text=Create%20a%20markup%20for%20the%20calendar%20block%20with%2031%20days%20inside) more carefully